Makefile

```diff
@@ -20,6 +20,7 @@ Payments: spec=PaymentService-v68
 Recurring: spec=RecurringService-v68
 Payouts: spec=PayoutService-v68
 Management: spec=ManagementService-v1
+Management: service=management
```

Comment:
Are we going to have two entries for every model and service per API. Is there another way to set this up?

Reply:
This is only necessary for services where we generate both models and the client.

Reply:
But that will probably be all of them in due time